process/coordinator/process.go

```diff
@@ -423,6 +423,16 @@ func (tc *transactionCoordinator) RemoveTxsFromPool(body *block.Body) error {
 	return errFound
 }
 
+// ForgetAllAccountNoncesInMempool instructs the mempool to forget all account nonces
+func (tc *transactionCoordinator) ForgetAllAccountNoncesInMempool() {
+	preproc := tc.getPreProcessor(block.TxBlock)
```

Comment:
Question for review: all right? We're only interested into the actual mempool (_source is me_).

Reply:
yep.